newspaper/network.py

```diff
@@ -80,8 +80,13 @@ def _get_html_from_response(response):
         # return response as a unicode string
         html = response.text
     else:
-        # don't attempt decode, return response in bytes
         html = response.content
+        if 'charset' not in response.headers.get('content-type'):
```

Comment:
Is this `if` statement needed? Since at this stage `response.encoding` is already empty or failed, we want to run `requests.utils.get_encodings_from_content` no matter what.

I think we should remove the if check and just keep
```
html = response.content
encodings = requests.utils.get_encodings_from_content(response.text)
if len(encodings) > 0:
  response.encoding = encodings[0]
  html = response.text
```

Reply:
Actually, I'm not sure, but suppose not.  This `if` is added because when the encoding is ISO-8895-1(which is the major case), the whole branch will be run, and I think the `get_encodings_from_content` maybe less efficient(a regular expression match?).

Reply:
I see, thanks